packages/adblocker/src/engine/domains.ts

```diff
@@ -12,11 +12,25 @@ import { StaticDataView, sizeOfUint32Array, sizeOfUTF8 } from '../data-view.js';
 import { binLookup, hasUnicode, HASH_INTERNAL_MULT } from '../utils.js';
 
 export class Domains {
-  public static parse(parts: string[], debug: boolean = false): Domains | undefined {
+  public static parse(
```

Comment:
lets replace this interface with `Domains.parse(parts, delimiter, debug)`

Reply:
fixed in [`95d53ed` (#4525)](https://github.com/ghostery/adblocker/pull/4525/commits/95d53ed76f128ed40623e94c6cc0e3a5375bbbbd)